apps/librelingo_audios/librelingo_audios/cli.py

```diff
@@ -0,0 +1,47 @@
+from pathlib import Path
+import click  # type: ignore
+import collections
+from librelingo_yaml_loader import load_course
+from librelingo_audios.generate_audios import generate_audios_for_course
+
+
+Settings = collections.namedtuple(
+    "Settings",
+    [
+        "dry_run",
+        "overwrite",
+    ],
+)
+
+DEFAULT_SETTINGS = Settings(
+    dry_run=False,
+    overwrite=False,
+)
+
+
+def ensure_output_directory(output_path, settings):
+    if not settings.dry_run:
+        Path(output_path).mkdir(parents=True, exist_ok=True)
+
+
+@click.command()
+@click.argument("input_path")
+@click.argument("output_path")
+@click.argument("course_name")
+@click.option("--dry-run/--no-dry-run", default=DEFAULT_SETTINGS.dry_run)
+@click.option("--overwrite/--no-overwrite", default=DEFAULT_SETTINGS.overwrite)
```

Comment:
I would put some `help=""` here explaining what this does, as the idea of downloading everything all over from scratch is a bit strong. Some people might believe that this would just _update_ things that need to be updated